### PR TITLE
Move to Keycloak 22.0.4

### PR DESCRIPTION
**Pull request by jurakp**
Updated required libraries
Adjusted JDBC Ping configuration (related to the PR submitted today https://github.com/mbta/terraform-keycloak-sso/pull/16)

Note: Since the base image is now ubi9-micro, there is no package manager at all.

### CODE REVIEW

File: files/conf/cache-ispn-jdbc-ping.xml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+        xsi:schemaLocation="urn:infinispan:config:14.0 http://www.infinispan.org/schemas/infinispan-config-14.0.xsd"
+        xmlns="urn:infinispan:config:14.0">
         
 	<jgroups>
 		<stack name="jdbc-ping-tcp" extends="tcp">
 			<JDBC_PING connection_driver="org.mariadb.jdbc.Driver"
 				connection_username="${env.KC_DB_USERNAME}" connection_password="${env.KC_DB_PASSWORD}"
-				connection_url="${env.KC_DB_URL}"
-				initialize_sql="CREATE TABLE IF NOT EXISTS JGROUPSPING (own_addr varchar(200) NOT NULL, cluster_name varchar(200) NOT NULL, ping_data BYTEA, constraint PK_JGROUPSPING PRIMARY KEY (own_addr, cluster_name));"
+				connection_url="jdbc:mysql://${env.KC_DB_URL_HOST}/${env.KC_DB_URL_DATABASE}?autoReconnect=true"
```
Comment (paulswartz) [inline files/conf/cache-ispn-jdbc-ping.xml]:
```suggestion
				connection_url="jdbc:mysql://${env.KC_DB_URL_HOST}/${env.KC_DB_URL_DATABASE}${env.KC_DB_URL_PROPERTIES}"
```

**Review by paulswartz — COMMENTED** [review]
I think the `docker-compose.yml` file also needs to be updated with the new DB environment variables.

**Review by krisrjohnson21 — APPROVED** [review]
🧼

**Comment by ianwestcott** [comment]
@jurakp at some point can you push the updated `mbta-keycloak-browser-authenticator` code to the https://github.com/mbta/keycloak-browser-authenticator repo?

**Pull request merged**